backend/notifications.py

```diff
@@ -22,6 +24,23 @@ def HandleNotification(firebase, notification_id, notification):
     firebase.put('/players/%s/notifications' % player,
                   notification_id, notification)
 
+  if 'app' in notification:
+    player_set = set(players)
```

Comment:
May as well just make players a set in the first place.
Avoid placing data types in var names.